bin/GameIndex.dbf

```diff
@@ -16331,6 +16452,16 @@ Name   = Sims 2, The - Pets
 Region = PAL-M11
 Compat = 5
 ---------------------------------------------
+Serial = SLES-54349
+Name   = Superman Returns
+Region = PAL-I
+[patches = E7F7B6BD]
+	author=kozarovv
+	// Use iaddiu instead of FSAND.
+	patch=1,EE,00639ef0,word,10050208
+	patch=1,EE,0063a068,word,10080208
+[/patches]
+---------------------------------------------
```

Comment:
See if koz can provide an actual explanation of what is going on. The comment is not descriptive.